recipes/dynamo-release/meta.yaml

```diff
@@ -0,0 +1,73 @@
+{% set name = "dynamo-release" %}
+{% set version = "1.3.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/aristoteleo/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 4b9e0b9721802b0b7baa05a0651518958d512660703ca0d62a5140a7689f3d22
+
+build:
```

Comment:
Can this be converted to a noarch recipe? https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-builds

Reply:
Thanks for the suggestion. I have added the noarch build.